src/perftest_resources.c

```diff
@@ -3286,9 +3291,6 @@ void ctx_set_send_reg_wqes(struct pingpong_context *ctx,
 
 			if ((j + 1) % user_param->cq_mod == 0) {
 				ctx->wr[i*user_param->post_list + j].send_flags = IBV_SEND_SIGNALED;
-				#ifdef HAVE_IBV_WR_API
```

Comment:
I don't understand why did you delete all of that...

Reply:
The send flags in post list mode should be determined per WR and not per list,
for example, to set IBV_SEND_SIGNALED according to the cq mod.
The extended QP wr_flags field should be set according to the WR flags before invoking each WR builder.

Reply:
Right, okay